debian/ceph-mon.postinst

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```

Comment:
So partly this is just satisfying Debian policy ( https://www.debian.org/doc/debian-policy/ch-files.html#s-scripts ) and bringing these scripts into line with all the other maintscripts that do correctly have a `#!` line. 
Is it a controversial change? It seems mostly self-evident to me that scripts should have a `#!` line...

Reply:
no, it is not.

was curious about its impact. for instance, why we haven't run into the issue. as the linux kernel should refuse to execute a script without shebang line. since, IIRC, debian / ubuntu maintainers have their own packaging recipes for ceph, we don't need to backport this change. the asking was just for posterity and for myself.

Reply:
could you please reference the Debian policy in the commit message?

Reply:
Debian's `ceph-mon.postinst` is essentially the same but with the `#!` added.
[in practice, I think a script without a `#!` will be treated as a series of commands by the current shell, which I suspect means no-one with e.g. `tcsh` as their default root shell has installed these packages directly from Ceph ]
I'm happy to update the commit message. This change is in one of the smaller PRs (#53397) - would you like me to make that fix there as well, and/or close that (and #53342 ?) in favour of this PR? I don't mind either way, but would rather avoid unnecessary work :)

Reply:
> Debian's `ceph-mon.postinst` is essentially the same but with the `#!` added.
> [in practice, I think a script without a `#!` will be treated as a series of commands by the current shell, which I suspect means no-one with e.g. `tcsh` as their default root shell has installed these packages directly from Ceph ]

i see. thank you. i have the impression that linux kernel interprets the shebang. but yeah, probably the `$SHELL` is also in the picture.

> I'm happy to update the commit message. This change is in one of the smaller PRs (#53397) - would you like me to make that fix there as well, and/or close that (and #53342 ?) in favour of this PR? I don't mind either way, but would rather avoid unnecessary work :)

ahh, sorry for the latency! since we have more discussion out here, in order to preserve the review history, maybe we should keep this PR and close the smaller ones. in general, we have smaller PRs because they are either 

- risky
- too large to review in a single batch
- involves different areas and should involve different reviewers interested these areas

but these do not apply to this changeset. because i think the risk is relatively low and the changes only touch the debian packaging. so probably you could just close the smaller PRs in favor of this one.

Reply:
Done.